src/app/SharpRaven/Utilities/Serializer.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.Serialization.Formatters.Binary;
+using System.Text;
+
+namespace SharpRaven.Utilities
+{
+    public static class Serializer
+    {
+        public static byte[] SerialiseBytes(this object obj)
```

Comment:
Since what's sent to Sentry is JSON, it **needs** to contain everything we need. So yes, I believe that's a better way to serialise `JsonPacket` than with `BinaryFormatter`. 😄